.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
     }
   },
   "ruby.format": "rubocop",
-  "ruby.intellisense": "rubyLocate"
+  "ruby.intellisense": "rubyLocate",
+  "eslint.enable": false
```

Comment:
not sure this should be disabled for everyone, what if you just disabled it in your user settings?

Reply:
Sure. Will disable it.